internal/cmd/local/command_kafka_stop.go

```diff
@@ -51,7 +51,7 @@ func (c *Command) kafkaStop(cmd *cobra.Command, args []string) error {
 			}
 			log.CliLogger.Tracef("Confluent Local container removed")
 
-			output.Printf("Stopped Confluent Local container %v.\n", getShortenedContainerId(container.ID))
+			output.Printf("Confluent Local has been stopped: removed container %v.\n", getShortenedContainerId(container.ID))
```

Comment:
I assume this is a string?
```suggestion
			output.Printf("Confluent Local has been stopped: removed container \"%s\".\n", getShortenedContainerId(container.ID))
```